Lib/_pylong.py

```diff
@@ -16,6 +16,9 @@
 import decimal
 
 
+_is_pydecimal = hasattr(decimal.Decimal, '_power_exact')
```

Comment:
```suggestion
_is_pydecimal = not hasattr(decimal, '__libmpdec_version__')
```

I think it's better to rely on that the C version will always have a "'__libmpdec_version__" attribute than on the names of internal `_pydecimal` functions.